source/db/dffml_source_db/source.py

```diff
@@ -0,0 +1,143 @@
+import collections
```

Comment:
I can't remember if I said to make this db plugin or not. So sorry if I did, but either way, we're going to go ahead and not have this be a plugin. Since it has no external dependencies.

So this `source/db/dffml_source_db/source.py` file should really go in `dffml/source/db.py`